main/src/libraries/wrdac/include/wrdac/subsystems/subSystem_KARMA.h

```diff
@@ -23,13 +23,18 @@
 #include <algorithm>
 #include <yarp/os/all.h>
 #include <yarp/sig/all.h>
+#include <yarp/dev/all.h>
 
 #include "wrdac/subsystems/subSystem.h"
 #include "wrdac/subsystems/subSystem_ABM.h"
 #include "wrdac/subsystems/subSystem_attention.h"
 
 #define SUBSYSTEM_KARMA       "KARMA"
 
+using namespace yarp::dev;
+using namespace yarp::os;
+using namespace yarp::sig;
```

Comment:
Please don't use namespaces in header files .. it's evil. Imagine someone includes this header file. Then, this poor guy doesn't know you force these namespaces upon him. So just use it in the cpp file, but not in the header ...